### PR TITLE
docs(README): `mangle.props` instead of `mangle.properties`

### PR DESCRIPTION
**Pull request by davilima6**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Bug Fixes
 
+* Fix typo: mangle properties key should be props
```
Comment (michael-ciniawsky) [inline CHANGELOG.md]: Please revert this change as the `CHANGELOG` is generated and the PR isn't part of this particular release :)

Comment (davilima6) [reply CHANGELOG.md]: Thanks for the advice but I think this PR is not correct (it changed from 0.4.6 to 1.x)

Comment (michael-ciniawsky) [reply CHANGELOG.md]: The `CHANGELOG` is still auto generated from commit messages and your PR isn't part of `v1.1.6` release, so this needs to be reverted please :)

Comment (davilima6) [reply CHANGELOG.md]: done

**Comment by kzc** [comment]
This PR is incorrect. Please revert.

The uglify `minify()` `mangle` sub-option is `properties`, not `props`.

See: https://github.com/mishoo/UglifyJS2/tree/harmony#minify-options-structure

**Comment by michael-ciniawsky** [comment]
https://github.com/webpack-contrib/uglifyjs-webpack-plugin/commit/44b77e3b2bec5d63baf340c382d6564123a4d108
